site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,117 @@
 title: 'Automatic annotation'
 linkTitle: 'Automatic annotation'
 weight: 16
-description: 'Guide to using the automatic annotation of tasks.'
+description: 'Automatic annotation of tasks'
 ---
 
-Automatic Annotation is used for creating preliminary annotations.
-To use Automatic Annotation you need a DL model that can be deployed by a CVAT administrator.
-You can find the list of available models in the `Models` section.
+Automatic annotation in CVAT is a tool that you can use
+to automatically pre-annotate your data with pre-trained models.
 
-1. To launch automatic annotation, you should open the dashboard and find a task which you want to annotate.
-   Then click the `Actions` button and choose option `Automatic Annotation` from the dropdown menu.
+CVAT can use models from the following sources:
+
+- [Pre-installed models](#models).
+- Models integrated from [Hugging Face and Roboflow](#adding-models-from-hugging-face-and-roboflow).
+- [Self-hosted models deployed with Nuclio](/docs/manual/advanced/serverless-tutorial/).
+
+The following table describes the available options:
+
+|                                             | Self-hosted            | Cloud                                            |
+| ------------------------------------------- | ---------------------- | ------------------------------------------------ |
+| **Price**                                   | Free                   | See [Pricing](https://www.cvat.ai/pricing/cloud) |
+| **Models**                                  | You have to add models | All models are pre-installed                     |
```

Comment:
"All models" doesn't seem like a correct description here. All of which ones?

IMO, it should say "You can use pre-installed models", which is accurate and also consistent with the left column.

Reply:
updated